src/main/java/envoy/client/data/Chat.java

```diff
@@ -111,6 +114,19 @@ public void insert(Message message) {
 		messages.add(0, message);
 	}
 
+	/**
+	 * Increments the unreadMessagesAmount by 1
```

Comment:
Incrementing something means increasing its value by one.

```suggestion
	 * Increments the amount of unread messages.
```